src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -204,11 +206,8 @@ public void onGuildVoiceLeave(GuildVoiceLeaveEvent event) {
                 UpdateCommand.update(true, null);
             }
         } else {
-            if (event.getMember().getUser().equals(event.getJDA().getSelfUser())) {
-                return;
-            }
-            if (event.getChannelLeft().getMembers().size() < 2 || event.getChannelLeft().getMembers()
-                    .stream().filter(m -> m.getUser().isBot()).count() == event.getChannelLeft().getMembers().size()) {
+            if (event.getChannelLeft().getMembers().contains(event.getGuild().getSelfMember()) && (event.getChannelLeft().getMembers().size() < 2 || event.getChannelLeft().getMembers()
```

Comment:
This is quite long, new line after the and before the start of the second comparison